core/primitives/scheduled_change.hpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_PRIMITIVES_SCHEDULED_CHANGE_HPP
+#define KAGOME_CORE_PRIMITIVES_SCHEDULED_CHANGE_HPP
+
+#include <boost/optional.hpp>
+#include "primitives/common.hpp"
+#include "primitives/session_key.hpp"
+
+namespace kagome::primitives {
+  /// @struct ScheduledChange is returned by Grandpa::pending_change function
+  struct ScheduledChange {
+    std::vector<std::pair<AuthorityId, uint64_t>> next_authorities_;
+    BlockNumber delay_;
```

Comment:
Please, name it without underscore in the end
https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
done